# I wonder who the hell chose such a docdir default without giving an easy
# way to override this... jweiner 06
diff -Naur libsndfile-1.0.15.old/configure.ac libsndfile-1.0.15/configure.ac
--- libsndfile-1.0.15.old/configure.ac	2006-04-29 12:25:38.000000000 +0200
+++ libsndfile-1.0.15/configure.ac	2006-04-29 12:26:49.000000000 +0200
@@ -390,13 +390,7 @@
 #====================================================================================
 # Settings for the HTML documentation.
 
-htmldocdir=$prefix/share/doc/libsndfile1-dev/html
-
-if test $prefix = "NONE" ; then
-	htmldocdir=/usr/local/share/doc/libsndfile1-dev/html
-else
-	htmldocdir=$prefix/share/doc/libsndfile1-dev/html
-	fi
+htmldocdir=$prefix/doc/libsndfile-1.0.15/html
 
 if test x$ac_arg_bow_docs = "xY" ; then
 	HTML_BGCOLOUR="white"
